lib/vdsm/tool/__init__.py

```diff
@@ -20,9 +20,39 @@
 from __future__ import absolute_import
 from __future__ import division
 import functools
+import logging
 import os
 
 
+LOGGER_NAME = "vdsm-tool"
+
+
+def setup_logging(log_file, verbosity, append):
+    level = logging.DEBUG if verbosity else logging.INFO
+    root_logger = logging.getLogger(LOGGER_NAME)
+    root_logger.setLevel(level)
+
+    if root_logger.handlers:
+        return root_logger
+
+    if log_file is not None:
+        mode = append and 'a' or 'w'
+        file_handler = logging.FileHandler(log_file, mode=mode)
+        file_handler.setFormatter(
+            logging.Formatter(
+                "%(asctime)s [%(levelname)-7s] %(message)s",
+                "%m/%d/%Y %I:%M:%S %p"))
+        # File logging always at DEBUG level
```

Comment:
Why?

Reply:
File logging is often used for reports and bug hunting, so default `DEBUG` level seemed appropriate. 

Taking into account that previously most messages were printed directly to stdout and not through logging, file logs were probably pretty empty. Now it will print all logs, which can be very useful in, e.g., an error occurred in the update-tool.

Reply:
OK, thanks for clarification.  But again, are we sure nothing relies on the current content of the file?

Please add the reasoning behind this change to the commit message.

Reply:
`are we sure nothing relies on the current content of the file?` No, that I am not sure, maybe @tinez knows?

I assume not, since, as I said, log file was probably pretty empty before. But there was some logging in network commands, so it could be that those were actually parsed.